capellambse_context_diagrams/filters.py

```diff
@@ -68,10 +68,10 @@ def uuid_filter(obj: common.GenericElement, label: str | None = None) -> str:
     str, cabc.Callable[[common.GenericElement, str | None], str]
 ] = {
     EX_ITEMS: lambda obj, _: exchange_items(obj),
-    FEX_EX_ITEMS: lambda obj, _: exchange_name_and_items(obj),
-    FEX_OR_EX_ITEMS: lambda obj, _: exchange_items(obj)
-    if obj.exchange_items
-    else obj.name,
+    FEX_EX_ITEMS: exchange_name_and_items,
+    FEX_OR_EX_ITEMS: lambda obj, label: exchange_items(obj)
+    if getattr(obj, "exchange_items", "")
```

Comment:
This `getattr()` based approach feels like more of a workaround around another bug than a real fix to me. How can it happen that `obj` is something that doesn't have `exchange_items` in the first place? Is it normal / expected that those objects appear there? And would it make more sense to filter out those objects before handing them over to the `FILTER_LABEL_ADJUSTERS`, or would that make other code unnecessarily complex?

Reply:
To be honest the usage of lambdas is solely because of the under-complexity of implementing the needed variants of `exchange_items`. As you can see almost all label-adjusters are about exchange items (except `NO_UUID`). Instead of adding normal functions (even shorter in body than the other ones `exchange_name_and_items`, `uuid_filter` which were lambdas at some point in time before too) where the boilerplate + docstring together would take up more lines than the logic we kept these as lambdas. If the user is using any `EX_ITEMS` filter on diagrams where the exchanges don't have this attribute then there is no effect instead of breaking. Also there are/might be diagramtypes where a mix of exchanges are apparent: Some with and w/o `.exchange_items`.

Reply:
So, to make it short: The `obj` is user input (or directly derived from it), and it's valid if it doesn't have `exchange_items`. That's all I wanted to be sure of :)